### ISSUE TITLE
How to discover new intents?

### ISSUE THREAD

**Issue by Ritali** [thuiar/CDAC-plus#3]
What the meaning of ”discover new intents“?
"we set BookRestaurant and SearchCreativeWork as unknown in training set. Our method is still able to ﬁnd out these intents."
As shown in the paper, the two unknown intents also need to know their label name? If there is a query with intent which is totally different from any known and unknown intents in the training data, which the predicted intent will be? It will be one in the training data?

**Comment by abhiishekpal** [comment]
No this isn't necessary, go through the data preperation part of dbpedia. The classes can just be numeric value

**Comment by HanleiZhang** [comment]
Q1) In this work, discovering new intents means finding new categories rather than new labels. It is because we do not know the true label of new intents in real-world scenarios. However, it is also valuable to discover new group of samples with the same intent, which is helpful to save time and human-labour to anotate the new label.
We hope to use known intent labeled data for learning pair-wise similarities and transfer the knowledge to group the unlabeled known and new intents. We only use the true labels for evaluating the performance of the clustering during testing. 

Q2) Each query will be assigned to its corresponding cluster by learning pair-wise similarities. We mainly focus on the meaningful intents with a group of samples, so a single anomalous sample does not matter.

Hope this answer is helpful to you.
